pyproject.toml

```diff
@@ -17,7 +17,7 @@ finufft = ["finufft"]
 pynfft = ["pynfft2", "cython<3.0.0"]
 pynufft = ["pynufft"]
 
-test = ["pytest", "pytest-cov", "pytest-xdist", "pytest-sugar", "pytest-cases"]
+test = ["pytest<8.0.0", "pytest-cov", "pytest-xdist", "pytest-sugar", "pytest-cases"]
```

Comment:
I think its better to ensure we support >8.0.0 right? Is it a big issue, maybe open one?

Reply:
Well https://github.com/smarie/python-pytest-cases/issues/330 is already there. 

I don't know (yet) how to remove the need for pytest-cases, we rely heavily on it for the parametrization...